strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/MavenProxyRepositoryProviderTestIT.java

```diff
@@ -168,15 +152,35 @@ public void shouldBeAbleToProvideFilesFromOracleMavenRepoWithHttpsAndAuthenticat
     }
     */
 
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class,
+            ArtifactManagementTestExecutionListener.class })
     @Test
-    public void whenDownloadingArtifactMetadaFileShouldAlsoBeResolved(TestInfo testInfo)
+    public void whenDownloadingArtifactMetadaFileShouldAlsoBeResolved(@MavenRepository(storageId = STORAGE_ID,
+                                                                                       repositoryId = REPOSITORY_ID + "-whenDownloadingArtifactMetadaFileShouldAlsoBeResolved")
+                                                                      Repository repository1,
+                                                                      @MavenRepository(storageId = STORAGE_ID,
+                                                                                       repositoryId = CENTRAL_REPOSITORY_ID + "-whenDownloadingArtifactMetadaFileShouldAlsoBeResolved")
+                                                                      Repository repository2,
+                                                                      @TestRepository(layout = LAYOUT_NAME,
+                                                                                      storageId = STORAGE_ID,
+                                                                                      repositoryId = REPOSITORY_ID + "-whenDownloadingArtifactMetadaFileShouldAlsoBeResolved",
+                                                                                      setup = MavenIndexedRepositorySetup.class)
+                                                                      @Remote(url = REMOTE_URL)
+                                                                      Repository proxyRepository1,
```

Comment:
You only need this repository `proxyRepository1` for this method. Remove the rest of them.
Please annotate it as `@MavenRepository` and remove the layout parameter.

Reply:
@ptirador Can you please take a look at the changes I made?